README.md

```diff
@@ -434,12 +572,23 @@ If you want to use an image in your file system:
 ```yaml
 collections:
   Jurassic Park:
-    tmdb_list: https://www.themoviedb.org/collection/328
+    tmdb_collection: https://www.themoviedb.org/collection/328
     tmdb_summary: 328
     file_poster: C:/Users/username/Desktop/2xE0R9I.png
     file_background: /config/backgrounds/Jurassic Park.png
 ```
 
+### System Name (Collection Attribute)
```

Comment:
The more I've thought about this, the less intuitive the name seems. What about like `file_name_override` or something? `system_name` could mean a lot of different things. Trying to think of something that is intuitive without the README

Reply:
I like file_name_override much better

Reply:
or `file_name_mapping`?

Reply:
what about just name_mapping

Reply:
switched to `name_mapping` easy to change if we don't end up going with it